### PR TITLE
Data Release for 2024-01-10

### PR DESCRIPTION
**Pull request by dwnoble**
Will fold in new places build once it's completed.

### CODE REVIEW

File: internal/server/v1/propertyvalues/golden/bulk_property_values_in/typeOf.json [hunk 1]
```diff
@@ -6690,7 +6690,7 @@
             "City"
           ],
           "dcid": "geoId/3602951803",
-          "provenanceId": "dc/base/WikidataOtherIdGeos"
+          "provenanceId": "dc/base/BaseGeos"
```
Comment (pradh) [inline internal/server/v1/propertyvalues/golden/bulk_property_values_in/typeOf.json:6693]: @n-h-diaz any ideas why this happens?

File: internal/server/v1/variables/golden/bulk_variables/california_and_santa_clara_union.json [hunk 2]
```diff
@@ -9977,19 +9971,7 @@
...
         "Count_Person_15OrMoreYears_HispanicOrLatino",
-        "Count_Person_15OrMoreYears_InternetUsageAtAnyLocation_IsInternetUser",
-        "Count_Person_15OrMoreYears_InternetUsageAtCafeteria_IsInternetUser",
-        "Count_Person_15OrMoreYears_InternetUsageAtHome_IsInternetUser",
-        "Count_Person_15OrMoreYears_InternetUsageAtPublicPlace_IsInternetUser",
-        "Count_Person_15OrMoreYears_InternetUsageAtSchool_IsInternetUser",
-        "Count_Person_15OrMoreYears_InternetUsageAtTraveling_IsInternetUser",
-        "Count_Person_15OrMoreYears_InternetUsageAtWork_IsInternetUser",
-        "Count_Person_15OrMoreYears_JobSearchUsage_IsInternetUser",
-        "Count_Person_15OrMoreYears_LaptopUser",
```
Comment (pradh) [inline internal/server/v1/variables/golden/bulk_variables/california_and_santa_clara_union.json:9992]: @ajaits @hareesh-ms This was from Commerce_NTIA, did we drop these SVs?

**Review by pradh — APPROVED** [review]
Barring a couple of questions, generally LGTM.

Please wait for approval from @ajaits and @jehangiramjad

File: internal/server/v1/page/golden/place_page/asm.Demographics.json [hunk 3]
```diff
@@ -180,15 +180,18 @@
...
-            "2011": 665196496,
-            "2012": 651507733,
-            "2013": 637792141,
-            "2014": 624030741,
-            "2015": 610192900,
-            "2016": 596465626,
-            "2017": 582840458,
-            "2018": 568902350,
-            "2019": 554781038
+            "2011": 665644371,
```
Comment (jehangiramjad) [inline internal/server/v1/page/golden/place_page/asm.Demographics.json:183]: These are coming from some OECD import (not WDI). I am not sure if this is expected?

**Review by jehangiramjad — APPROVED** [review]
Thanks! WDI diffs look good to me. There is one change I saw which is coming from the OECD import and I am not sure whether that's expected or not.

**Comment by dwnoble** [comment]
Updated place bt table. No new changes to the golden file diff with that update.

File: internal/server/v1/variables/golden/bulk_variables/california_and_santa_clara_union.json (hunk 2 above)
Comment (ajaits) [reply internal/server/v1/variables/golden/bulk_variables/california_and_santa_clara_union.json:9992]: Yes, this is expected as `armedForcesStatus: dcid:Civilian` was added to convert the SVs to Count_Person_15OrMoreYears_**Civilian**_*

**Pull request merged**